### DISCUSSION TITLE
Turn this in to a rojo project

### DISCUSSION THREAD

**Discussion by ccuser44** [ExperiencersInternational/auroraSuite#5] [Feature Requests]
Rojo projects are easier to edit in Github.

**Comment by ghost** [comment]
This resource does not run at runtime, so it won’t work :/

**Comment by ExperiencersInternational** [comment]
> This resource does not run at runtime, so it won’t work :/

Surely it should still work? I did leave a response in a pull request stating that it probably wouldn't happen.

I know the plugin is like 1MB when it could be like 15KB or something, but I like being able to edit UI visually rather via Luau.
